MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -268,3 +276,20 @@ extension PassiveLocationDataSource {
         public static let roadNameKey: NotificationUserInfoKey = .init(rawValue: "roadName")
     }
 }
+
+extension MapboxNavigationService {
+    /**
+     Keys in the user info dictionaries of various notifications posted by instances of `NavigationService`.
+     */
+    public struct NotificationUserInfoKey: Hashable, Equatable, RawRepresentable {
+        public typealias RawValue = String
+        public var rawValue: String
+        public init(rawValue: String) {
+            self.rawValue = rawValue
+        }
+    }
+
+    /**
+     A key in the user info dictionary of a `Notification.Name.locationAuthorizationDidChange` notification. The corresponding value is a `Int` object the current location authorization setting. */
+    public static let locationAuthorizationKey: NotificationUserInfoKey = .init(rawValue: "locationAuthorization")
```

Comment:
This constant should be part of `MapboxNavigationService.NotificationUserInfoKey`, so that you’d use it as `MapboxNavigationService.NotificationUserInfoKey.locationAuthorizationKey` (or just `NotificationUserInfoKey.locationAuthorizationKey` if inside the implementation of MapboxNavigationService).